src/OpenSpaceToolkit/Physics/Environment/Ephemerides/SPICE/Engine.cpp

```diff
@@ -234,7 +234,7 @@ Array<Kernel> Engine::DefaultKernels(const Directory& aLocalRepository)
 
     static const Array<Kernel> defaultKernels = {
 
-        Manager::Get().findKernel("naif[0-9]*\\.tls"),                                    // Leap seconds
+        Manager::Get().findKernel("latest_leapseconds.tls"),                              // Leap seconds
```

Comment:
This file is available from the same URL as the other files and is updated at the source to always be latest. Better than regex IMO